faw/pdf-etl-parse/main.py

```diff
@@ -82,9 +82,21 @@ def handle_clean(conn_resolver, db_dst):
 
 def handle_doc(doc, conn_resolver, *, db_dst, fname_rewrite, parse_version,
         parsers_config, parser_parsers_shared):
-    """Parse this document."""
+    """Parse this document, and update the DB table `db_dst`."""
     conn_dst = conn_resolver(db_dst)
+    d = parse_features(
+        doc,
+        fname_rewrite=fname_rewrite,
+        parse_version=parse_version,
+        parsers_config=parsers_config,
+        parser_parsers_shared=parser_parsers_shared,
+    )
+    conn_dst.replace_one({'_id': doc['_id']}, d, upsert=True)
 
+
+def parse_features(doc, *, fname_rewrite, parse_version,
```

Comment:
Split this off into a second top-level function for use in `faw_analysis_set_parse.py`. The interface to `handle_doc` isn't changed.